pom.xml

```diff
@@ -4,18 +4,19 @@
 
     <groupId>me.totalfreedom</groupId>
     <artifactId>totalfreedom</artifactId>
-    <version>5.0</version>
+    <version>5.1</version>
```

Comment:
Considering this is still TFM Electrum, and not TFM _F_. We should stay at `5.0`.

Reply:
Yes, I think I got some of the versioning confused over the last few months. Shall fix this.